platform/golden/cfg/debian-11-preseed.cfg

```diff
@@ -0,0 +1,92 @@
+####################
```

Comment:
Leaving this as a breadcrumb for code archeologists:
https://wiki.debian.org/DebianInstaller/Preseed